script-test/playbackstrategies/msestrategytest.js

```diff
@@ -1,491 +1,579 @@
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=45', true);
+            mseStrategy.load('src2', null, 0);
 
-          mseStrategy.load('src2', null, 86);
+            expect(mockDashInstance.attachSource).toHaveBeenCalledWith('src2#t=45');
 
-          expect(mockDashInstance.attachSource).toHaveBeenCalledWith('src2#t=86');
-        });
+            mseStrategy.load('src3', null, 0);
 
-        it('should playback from relative live start time for simulcast', function () {
-          setUpMSE(0, WindowTypes.SLIDING, MediaKinds.VIDEO);
+            expect(mockDashInstance.attachSource).toHaveBeenCalledWith('src3#t=45');
+          });
 
-          mockDashInstance.getSource.and.returnValue('src');
+          it('should correctly continue playback from reached time when CDN failover occurs', function () {
+            setUpMSE();
 
-          // Actual live point requested
-          mseStrategy.load('src', null, 0);
+            mockDashInstance.getSource.and.returnValue('src');
+            mockVideoElement.currentTime = 86;
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#r=-1', true);
-        });
+            mseStrategy.load('src', null, 45);
 
-        it('should playback from relative start time for simulcast', function () {
-          setUpMSE(0, WindowTypes.SLIDING, MediaKinds.VIDEO);
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=45', true);
 
-          mockDashInstance.getSource.and.returnValue('src');
+            mseStrategy.load('src2', null, 86);
 
-          // Somewhat live, playbackUtils forces 0.1 for TAL related reasons!
-          mseStrategy.load('src', null, 0.1);
+            expect(mockDashInstance.attachSource).toHaveBeenCalledWith('src2#t=86');
+          });
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#r=0', true);
-        });
+          it('should playback from relative live start time for video simulcast', function () {
+            setUpMSE(0, WindowTypes.SLIDING, MediaKinds.VIDEO);
 
-        it('should playback from derived start time for webcast', function () {
-          setUpMSE(0, WindowTypes.GROWING, MediaKinds.VIDEO, 100000, 200000);
+            mockDashInstance.getSource.and.returnValue('src');
 
-          mockDashInstance.getSource.and.returnValue('src');
+            // Actual live point requested
+            mseStrategy.load('src', null, 0);
 
-          mseStrategy.load('src', null, 0.1);
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#r=-1', true);
+          });
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=101', true);
-        });
+          it('should playback from relative live start time for audio simulcast', function () {
```

Comment:
I don't think this test is necessary as the only variance is the initialize should be called with an audio element, but this is already tested in the above new test.